rpmlb/downloader/base_rpkg.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from .. import utils
+from .base import BaseDownloader
+
+LOG = logging.getLogger(__name__)
+
+
+class BaseRpkgDownloader(BaseDownloader):
+    """A downloader class to get a pacakge with package command."""
+
+    def command(self):
```

Comment:
This should be a `@property`.

I would envision the whole inheritance tree to look something like this:

```python
class Base:
    @property
    def command(self):
        raise NotImplementedError('...')

    def echo_command(self):
         """Example method that uses the command property."""
        print(self.command)


class A(Base):
    command = 'rhpkg'


class B(Base):
    command = 'fedpkg'
```
```python
>>> a, b = A(), B()
>>> a.echo_command()
rhpkg
>>> b.echo_command()
fedpkg
```